src/Components/Buttons/Styles.js

```diff
@@ -0,0 +1,37 @@
+const Styles = {
```

Comment:
Changes this to lower case.  Also each component should have it's own directory with it's own styles.js file (with a Stylesheet in it).

 You can group them in Buttons directory but, it is not the way we usually do it . In general, in the projects we have two directories: `components/screens` and` components/common`.

Reply:
Done!